.changeset/neat-cameras-share.md

```diff
@@ -0,0 +1,5 @@
+---
+'@stacks/wallet-web': patch
+---
+
+Fixes issue where the fee wasn't subtracted from the maximum amount you can send
```

Comment:
I'm hoping we can start using a bit more verbose updates here. I think the more time we spend here, the better the experience will be for folks who come to check out what changed.

Reply:
Can you suggest a friendlier alternative?